app/delivery/send_to_providers.py

```diff
@@ -40,10 +41,12 @@ def send_sms_to_provider(notification):
     if notification.status == 'created':
         provider = provider_to_use(SMS_TYPE, notification.international)
 
-        template_model = dao_get_template_by_id(notification.template_id, notification.template_version)
+        template_dict = SerialisedTemplate.get_dict(template_id=notification.template_id,
```

Comment:
Could we use `from_id_and_service_id` instead? It looks like that may be even more efficient, due to `@memory_cache`.